Sorts/HeapSort.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+
+void heapify(int *unsorted, int index, int heap_size);
+void heap_sort(int *unsorted, int n);
+
+int main() {
+	int n = 0;
+	int i = 0;
+	char s[50];
+	char oper;
+
+	int* unsorted;
+	printf("Enter the size of the array you want\n");
+	scanf("%d", &n);
+	unsorted = (int*)malloc(sizeof(int) * n);
+	while (getchar() != '\n');
```

Comment:
Keep it simple! Input integers directly rather than using atoi for casting.

Reply:
Are you saying that you want to initialize the array and implement it?